### DISCUSSION TITLE
Because You Are the Tag King! 😺

### DISCUSSION THREAD

**Discussion by kertripmerth** [windingwind/zotero-better-notes#755] [General]
Hi Windingwind,

In the PDF editor of the current version of Zotero the tags that users can add to the documents are not as effective as they could be.

For me, there seem to be two big issues.

The tag area itself is not re-sizeable, it's fixed at like just 15% of the lower side of the left column.

And, the tags themselves are not alphabetically ordered.  They are ordered only by the latest tag made.

Once you get past a certain number of tags, the system is almost unusable.  You'll need to REMEMBER every tag you ever made, because you can't see them.

Worse yet, you have to remember WHEN you made them, because there's no sorting except by when you made the tag.

Do you think you could write a plugin that could fix these two issues?

The idea is: to allow the tags area to be re-sized.

And, to alphabetize the tags.  (If you want to get fancy, even a number count after each tag / sorting by number. 😃 )

I'm sure you have folks begging you for fixes all the time, and sadly, I don't have any funds I can offer, but I can promise you, these are two **innovations that would be WARMLY welcomed.**

The tags as they are now in the PDF viewer are kind of impossible to put to use.

Kindest regards,

A BIG Fan of Windingwind

**Comment by windingwind** [comment]
Generally, this is out of the scope of plugins and is more suitable to be discussed in the official forums. Only when the team explicitly refuses to consider doing this, plugins are suitable to interfere.

For me, I do not have more time available for any new plugins. If anyone is interested, they can take it, but that would not be me - I'm too busy.

Thanks for the suggestion. I will close this discussion because it is off-topic in this repo.
